### ISSUE TITLE
Crossorigin="anonymous" since update Contao 4.7

### ISSUE THREAD

**Issue by Kahmoon** [contao/contao#356]
**Affected version(s)**
4.7

**Description**  
Due to some changes in 4.7 for example google maps doesn´t work anymore. Maybe other modules are affected too.

`<script src="https://maps.googleapis.com/maps/api/js?key=XYZ&language=de"></script>`

became

`<script src="https://maps.googleapis.com/maps/api/js?key=XYZ&language=de" crossorigin="anonymous"></script>`

**Error:**
`Access to script at 'https://maps.googleapis.com/maps/api/js?key=XYZ&language=de' from origin 'https://mein-hostname' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

**There is also a (new) setting in .htaccess**
```
<IfModule mod_headers.c>
        # Assets in /assets and /bundles either contain a hash in their filename
        # or are called with a ?version suffix, therefore cache them for 1 year.
        RewriteRule ^(assets|bundles)/ - [ENV=CONTAO_ASSETS:true]
        Header set Cache-Control "max-age=31536000" env=CONTAO_ASSETS


        # Allow CORS on the Contao TinyMCE skin.
        RewriteRule ^assets/tinymce4/js/skins/contao/fonts/ - [ENV=CONTAO_TINYMCE_SKIN:true]
        Header set Access-Control-Allow-Origin "*" env=CONTAO_TINYMCE_SKIN
    </IfModule>
```

Greetings Frank

**Comment by fritzmg** [comment]
This was added in https://github.com/contao/contao/commit/65a878ddbd7e156c1e1dd1056cb900d2d5ac2e28

**Comment by leofeyer** [comment]
Fixed in a065797c9c0d9a03defee79f8c66ef3c6e5d964c.
